src/main/js/dev/rest/localisation/translations_sv.json

```diff
@@ -402,11 +344,12 @@
   "registryItem.deleteCancel": "Ta inte bort anordnare",
   "registryItem.deleteConfirm": "Bekräfta radering",
   "registryItem.extra": "Ytterligare information",
-  "registryItem.extra.placeholder": "Till exempel Allmän e-postlista alla@anordnare",
+  "registryItem.extra.placeholder": "Till exempel Allmän e-postlista alla@anordnare.fi",
```

Comment:
"T.ex" instead of "Till exempel" probably works too. Also "Allmän" can be without caps, and `.fi` is yet to be added to the email address in untuva.